internal/telemetry/config.go

```diff
@@ -27,7 +27,7 @@ func NewUUID() UUID {
 
 // String returns a string representation of this UUID.
 func (u UUID) String() string {
-	return uuid.UUID(u).String()
+	return u.ToUUID().String()
```

Comment:
Why do we need to call `.ToUUID()` before calling `.String()` and not just call `.String()` directly like we do in `UnmarshalYAML()` below?

Reply:
Sorry, I'm mixing up the semantics of type embedding and aliasing. `UUID` our defined type does have access to the referent type's methods